Migrating.md

```diff
@@ -1,6 +1,12 @@
 Migrating from previous versions
 ================================
 
+Migrating from 7.x to 8.x versions
```

Comment:
Well, assume this is a preparation for the next major release :)

But in fact, you're a bit wrong. Let me explain in detail:

* we mark `.options()` as deprecated in the current (v7) version (ideally we should do this with the initial v7 release, but I think now is ok, as it was released just a few weeks ago).
* As our new methods and the existing one are backward compatible — there is no need to upgrade the major version.
* To prepare library users for a new release (without `.options()`). We should describe this in the current documentation for `.options()` method.